microscopium/screens/cellomics.py

```diff
@@ -147,6 +147,9 @@ def snail_stitch(fns):
     ----------
     fns : list of array, shape (M, N)
```

Comment:
Wait, list of array? List of string???

In fact, this whole docstring needs overhauling. 

Regarding how to specify orderings, how about a dictionary, with named orderings? I suggest finding snail species of different sizes for the names. =P Then you can pass either a string (naming the ordering) or a list of lists, as you specify now. If string, it draws the order from `snail_orderings[order]`.